src/main/java/frc/robot/controllers/RobotController.java

```diff
@@ -45,12 +54,17 @@ public RobotController () {
         leftDriveMotor2 = new TalonFX(Context.leftMotor2ID);
         rightDriveMotor1 = new TalonFX(Context.rightMotor1ID);
         rightDriveMotor2 = new TalonFX(Context.rightMotor2ID);
-        rollingIntake = new TalonSRX(-1);
-        beltIntake = new TalonSRX(-1);
-        leftDriveEncoderInterface = new TalonSRX(Context.leftEncoderInterfaceID);
-        rightDriveEncoderInterface = new TalonSRX(Context.rightEncoderInterfaceID);
+
+        omniNeo = new CANSparkMax(Context.omniSparkID, MotorType.kBrushless);
+        NMFTalon = new TalonSRX(Context.indexerTalonID);
+        intakeTalon = new TalonSRX(Context.intakeMotorId);
+    
+        //Encoders
+        NMFEncoder = new Encoder(Context.indexerEncoderChannelA, Context.indexerEncoderChannelB);
```

Comment:
Encoder object can probably be moved to the NMF class. I think the motors are declared outside for testing purposes, but it will get too cluttered if every motor and sensor is in robotController.

Reply:
Maciej told to me to put it outside, because we won't be able to use the encoder in tests otherwise. Also the other encoders that are built in, like the neo and talon ones, are part of the motor controller and are in robotcontroller, so I guess it is more consistent having the rest of the encoders in robotcontroller too.